packs/scripts/run-migration.ts

```diff
@@ -219,6 +222,9 @@ async function migrate() {
                     }
                     for (const updatedItem of updatedActor.items) {
                         delete (updatedItem.system as { schema?: unknown }).schema;
+                        if (updatedItem.type === "consumable" && !updatedItem.system.spell) {
+                            delete (updatedItem.system as { spell?: unknown }).spell;
```

Comment:
This only deletes `updateItem.system.spell` if it doesn't exists on the item which seems redundant or am I reading that wrong?

Reply:
It deletes it if it's `null`